### PR TITLE
Mercury tests fix

### PR DESCRIPTION
**Pull request by tomasznapieralski**
Changes summary:
- change selectors name
- add cache buster to url (?cb=timestamp)
- combine two method for navigating through wiki on Mercury (using urlBuilder)
- add wait method for comments test
- add temporary solution for [HG-612](https://wikia-inc.atlassian.net/browse/HG-612) issue

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/ArticlePageObject.java [hunk 1]
```diff
@@ -29,7 +29,7 @@
   private List<WebElement> footerLinks;
   @FindBy(css = "div.contributors a")
   private List<WebElement> topContributorsLinks;
-  @FindBy(css = "nav.article-categories-list button")
+  @FindBy(css = "nav.article-categories-list div")
```
Comment (RodriGomez) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/ArticlePageObject.java]: Can you give more background on why the selector change? ;)

Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/ArticlePageObject.java]: HG team applied style guide and they were forced to change some selectors

**Comment by RodriGomez** [comment]
LGTM

**Comment by tomasznapieralski** [comment]
I will merge it when mercury@dev will be fixed

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/TOCTests.java [hunk 2]
```diff
@@ -26,10 +27,14 @@ public void prepareTest() {
   private final static int H2_PADDING_TOP = 40;
 
   // TOCT01
+  @RelatedIssue(issueID = "HG-612")
   @Test(groups = {"MercuryTOCTest_001", "MercuryTOCTests", "Mercury"})
   public void MercuryTOCTest_001_TOCPresence_ListRedirection() {
     BasePageObject base = new BasePageObject(driver);
     base.openMercuryArticleByName(wikiURL, MercuryArticles.MERCURY_TOC_TEST_ARTICLE);
+    // Temporary solution
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/TOCTests.java]: This line should be removed after HG-612 is done?

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/BasePageObject.java [hunk 3]
```diff
@@ -25,17 +27,12 @@ public BasePageObject(WebDriver driver) {
   }
 
   public ArticlePageObject openMercuryArticleByName(String wikiURL, String articleName) {
-    getUrl(wikiURL + URLsContent.WIKI_DIR + articleName);
+    getUrl(wikiURL + URLsContent.WIKI_DIR + articleName + "?cb=" + new Timestamp(System.currentTimeMillis()).getTime());
```
Comment (ludwikkazmierczak) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/BasePageObject.java]: Please use Yoda 
```DateTime.now().getMillis()``` for a timestamp

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/TOCTests.java (hunk 2 above)
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/TOCTests.java]: yes

**Pull request merged**